app/scraper.py

```diff
@@ -8,51 +8,46 @@
 import os
 
 import dataset
-import json
 import tweepy
 from sqlalchemy.exc import ProgrammingError
 from dotenv import load_dotenv
 import psycopg2
 from app.helper_funcs import getRankOfForce, clean_data
-
+from app.TagMaker import TagMaker
+from app.TagList import pb_tags
```

Comment:
I would try to add this import to the TagMaker Class file to clean up a little bit.